File: API/src/Models/Booking.ts
export interface Booking {
  id: string;                 
  vehicleId: string;          
  customerFirstName: string;
  customerLastName: string;
  customerEmail: string;
  customerLicenseId: string;
  customerMobile: string;
  startDate: number;          
  endDate: number;            
  keyStatus: "Not Retrieved" | "Retrieved" | "Dropped Off";
  dropoffCode: string;
  collectionCode: string;
}

File: CarRental/Models/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CarRental.Models
{
    public class Booking
    {
        public string Id { get; set; }
        public string VehicleId { get; set; }
        public string CustomerFirstName { get; set; }
        public string CustomerLastName { get; set; }
        public string CustomerEmail { get; set; }
        public string CustomerLicenseId { get; set; }
        public string CustomerMobile { get; set; }
        public long StartDate { get; set; }
        public long EndDate { get; set; }
        public string KeyStatus { get; set; }
        public string DropoffCode { get; set; }
        public string CollectionCode { get; set; }
    }

}
